.github/workflows/test-wheel-linux.yml

```diff
@@ -221,6 +230,21 @@ jobs:
           pushd ./cuda_core
           pip install -r "tests/requirements-cu${TEST_CUDA_MAJOR}.txt"
           pytest -rxXs tests/
+
+          # It is a bit convoluted to run the Cython tests against CTK wheels,
+          # so let's just skip them. Also, currently our CI always installs the
+          # latest bindings (from either major version). This is not compatible
+          # with the test requirements.
+          if [[ "${{ inputs.local-ctk }}" == 1 && "SKIP_CUDA_CORE_CYTHON_TEST" == 0 ]]; then
+            pip install cython
+            if [[ "${{ inputs.host-platform }}" == linux* ]]; then
+              bash tests/cython/build_tests.sh
+            elif [[ "${{ inputs.host-platform }}" == win* ]]; then
+              # TODO: enable this once win-64 runners are up
+              exit 1
+            fi
+            pytest -rxXs tests/cython
```

Comment:
Here and for the existing pytest calls: Add `-v`? (I often find that very useful.)

Reply:
I have slight preference over not cluttering the CI logs.

Reply:
In cases where we cannot easily reproduce locally, this can make the difference between day a night. (I've been there several times in my career.)

Extra log output is easy to ignore.

And often impossible to guess.

Reply:
OK, added in commit e4cde77